src/main/resources/application.yml

```diff
@@ -25,8 +25,9 @@ rest-source-authorizer:
 
   validator: managementportal
   management-portal:
-    base-url: "http://localhost:8081"
-    projects-path: "/projects"
-    subjects-path: "/subjects"
+    base-url: "http://localhost:8081/"
+    projects-path: "api/projects"
```

Comment:
ok. Now I get it. I think this doesn't necessarily need to be a config. 
Paths in ManagementPortal won't change that easily. This is fine for now, since this doesn't need to be configured by user. Otherwise, this will make the configuration process rather difficult.